app/poros/food.rb

```diff
@@ -0,0 +1,15 @@
+class Food
+  attr_reader :upc,
+              :description,
+              :brand_owner,
+              :ingredients,
+              :total_results
+
+  def initialize(json, total_results)
+    @upc = json[:gtinUpc]
+    @description = json[:description]
+    @brand_owner = json[:brandOwner]
+    @ingredients = json[:ingredients]
+    @total_results = total_results
```

Comment:
So the total_results number doesn't belong in an individual food object. That breaks SRP and the idea of encapsulation.

Reply:
Just saw your comment from the form that said you ran out of time and wanted to take totalResults out of this object. Glad to see you recognized this.